packages/radix-ui-themes/src/components/code.css

```diff
@@ -1,3 +1,6 @@
+/* stylelint-disable selector-max-type */
+/* Disable selector-max-type rule to target individual element types. */
```

Comment:
Getting close to disabling this rule in the global config and relying on a verbal convention not to use type selector without `:where`